### PR TITLE
OPENEUROPA-1659: Change display of base field.

### PR DESCRIPTION
**Pull request by imanoleguskiza**
### Description

Changed display of base fields to hide them from display.

### Change log

- Added:
- Changed: Hid base fields.
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
         "drupal/core": "^8.6",
         "drupal/field_group": "^1.0",
         "easyrdf/easyrdf": "0.10.0-alpha.1 as 0.9.1",
-        "openeuropa/rdf_skos": "dev-master",
+        "openeuropa/rdf_skos": "dev-OPENEUROPA-1659",
```
Comment (upchuk) [inline composer.json]: Back to dev-master

File: modules/oe_content_page/oe_content_page.install [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the OE Paragraphs module.
```
Comment (upchuk) [inline modules/oe_content_page/oe_content_page.install]: Copy pasta

File: modules/oe_content_page/oe_content_page.install [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+
+
```
Comment (upchuk) [inline modules/oe_content_page/oe_content_page.install]: too many empty lines

File: modules/oe_content_page/oe_content_page.install [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+/**
+ * Hide fields from page content type.
```
Comment (upchuk) [inline modules/oe_content_page/oe_content_page.install]: Let's add a bit more descriptive comment.

File: oe_content.install [hunk 5]
```diff
@@ -13,4 +13,25 @@ function oe_content_install() {
   if (!\Drupal::service('config.installer')->isSyncing()) {
     \Drupal::service('oe_content.op_skos_setup')->setup();
   }
-}
\ No newline at end of file
+}
+
+/**
+ * Hide base fields.
```
Comment (upchuk) [inline oe_content.install]: Let's add a bit more descriptive comment.

File: composer.json (hunk 1 above)
Comment (voidtek) [reply composer.json]: Done.

File: modules/oe_content_page/oe_content_page.install (hunk 2 above)
Comment (voidtek) [reply modules/oe_content_page/oe_content_page.install]: done.

File: modules/oe_content_page/oe_content_page.install (hunk 3 above)
Comment (voidtek) [reply modules/oe_content_page/oe_content_page.install]: done.

File: modules/oe_content_page/oe_content_page.install (hunk 4 above)
Comment (voidtek) [reply modules/oe_content_page/oe_content_page.install]: done.

File: oe_content.install (hunk 5 above)
Comment (voidtek) [reply oe_content.install]: done.

File: tests/features/news.feature [hunk 6]
```diff
@@ -30,8 +30,6 @@ Feature: News content creation
     And I fill in "Short title" with "Shorter title"
     When I press "Save"
     Then I should see "My News item"
-    And I should see "Navi title"
-    And I should see "Shorter title"
```
Comment (voidtek) [inline tests/features/news.feature]: If we configure this field hidden, we should check it, as ` And I should not see "Shorter title"`, isn't it?

Comment (upchuk) [reply tests/features/news.feature]: Not really necessary I guess. It's standard behaviour...

Comment (brummbar) [reply tests/features/news.feature]: I actually agree with @voidtek on this one 😃

Comment (brummbar) [reply tests/features/news.feature]: Let's convert this to "I should not see" 😄

File: tests/features/page.feature [hunk 7]
```diff
@@ -19,13 +19,7 @@ Feature: Page content creation
     And I fill in "Author" with "European Patent Office"
     When I press "Save"
     Then I should see "My page"
-    And I should see "Navi title"
-    And I should see "Shorter title"
-    And I should see "Summary text"
     And I should see "Body text"
-    And I should see "Teaser text"
-    And I should see the link "financing"
-    And I should see the link "European Patent Office"
```
Comment (brummbar) [inline tests/features/page.feature]: Same here.

**Pull request merged**